### PR TITLE
added max new lines option, fixes #125

### LINKED ISSUE #125: Add configuration option for setMaxNewlines

### PR DESCRIPTION
**Pull request by helios-ag**

### CODE REVIEW

File: Decoda/DecodaManager.php [hunk 1]
```diff
@@ -443,7 +443,7 @@ private function set($filterSet, Decoda $decoda = null)
         $decoda->setStrict($options['strict']);
         $decoda->setEscaping($options['escaping']);
         $decoda->setConfig(array('lineBreaks' => $options['line_breaks']));
-
+        $decoda->setMaxNewlines($options['max_newlines']));
```
Comment (alquerci) [inline Decoda/DecodaManager.php]: Oh! too many close parentheses.
Why remove the newline?

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -77,10 +77,11 @@ public function getConfigTreeBuilder()
...
                         ->children()
                             ->scalarNode('locale')->defaultValue('default')->end()
-                            ->booleanNode('xhtml')->defaultValue(true)->end()
-                            ->booleanNode('strict')->defaultValue(true)->end()
-                            ->booleanNode('escaping')->defaultValue(true)->end()
-                            ->booleanNode('line_breaks')->defaultValue(true)->end()
+                            ->booleanNode('xhtml')->defaultTrue()->end()
+                            ->booleanNode('strict')->defaultTrue()->end()
+                            ->booleanNode('escaping')->defaultTrue())->end()
+                            ->booleanNode('line_breaks')->defaultTrue())->end()
```
Comment (alquerci) [inline DependencyInjection/Configuration.php]: No needed to change these methods calls to add this feature.

**Comment by alquerci** [comment]
@helios-ag If I have time I will try to provide test cases for this if you do not.

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (helios-ag) [inline DependencyInjection/Configuration.php]: Agreed, but i think boolean nodes should have default*() setters. This changes doesn't relate to newline setting.

File: Decoda/DecodaManager.php (hunk 1 above)
Comment (helios-ag) [inline Decoda/DecodaManager.php]: Didn't realise this, will fix it later. My bad.

**Pull request merged**